### PR TITLE
Add light-blue-background addons to Scratch Addons

### PR DESCRIPTION
**Pull request by era2010-coder**
Add light-blue-background addons to Scratch Addons

### CODE REVIEW

File: addons/light-blue-background/addon.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Light blue background",
+    "description": "Adds a light blue background to Scratch",
+    "tags": ["community","popup"],
```
Comment (FunctionalMetatable) [inline addons/light-blue-background/addon.json:4]: not a popup

**Review by FunctionalMetatable — COMMENTED** [review]
too small to be an addon imo

File: addons/light-blue-background/addon.json (hunk 1 above)
Comment (era2010-coder) [reply addons/light-blue-background/addon.json:4]: ok sorry

Comment (era2010-coder) [reply addons/light-blue-background/addon.json]: ok there

**Comment by WorldLanguages** [comment]
Sorry we cannot accept addons like this. But you are free to use this addon in your local instance.

**Pull request closed without merging**